File: PrimeiroFormulario/app/src/main/java/com/example/primeiroformulario/MainActivity.java
package com.example.primeiroformulario;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    EditText edtNome;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        edtNome = findViewById(R.id.edtNome);
    }

    public void enviar(View view) {
        String nome = edtNome.getText().toString();
        if (nome.trim().isEmpty() ){
            Toast.makeText(this, "Informe o nome corretamente! ", Toast.LENGTH_LONG).show();
        }else {
            Toast.makeText(this, "Olá " + nome +". Seja Bem vindo(a)!", Toast.LENGTH_LONG).show();
        }

    }
}

File: TransicaoDoJavaParaKotlin/app/src/main/java/com/example/transicaodojavaparakotlin/MainActivity.kt
package com.example.transicaodojavaparakotlin

import android.support.v7.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {

    var nome : String? = null
    var contador : Int = 0
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var pessoa = Pessoa(nome:"Priscila", email:"prii.mfurlani@gmail.com")
        var pessoa

    }

    fun soma(a:Int, b:Int):Int{
        return a + b
    }

}
class Pessoa(var nome:String, var email:String)